guides/source/developers/extensions/testing-extensions.html.md

```diff
@@ -7,47 +7,67 @@ Solidus extensions.
 
 [contrib]: https://github.com/solidusio-contrib
 
-## TravisCI
+## CircleCI
 
-We usually test our extensions on TravisCI and make use of their [build matrix
-feature][build-matrix] to test across multiple Solidus versions. We also test
-against multiple databases: MySQL and PostgreSQL.
+We usually test our extensions against multiple Solidus versions and databases:
+MySQL and PostgreSQL. We use a [CircleCI Orb][orb] to re-use a maintain a single
+configuration that will be executed against all Solidus extensions.
 
-Solidus extensions should be tested across all versions of Solidus that have not
-reached [End of Life](https://solidus.io/blog/2018/01/04/maintenance-eol-policy.html)
-including the `master` branch.
+This Orb will take care of testing extensions against all versions of Solidus
+that have not reached [End of Life][eol] including the `master` branch.
 
-Here's an example `.travis.yml` testing versions 2.2 through 2.7 and the
-`master` branch:
+Here's an example `.circleci/config.yml` file that you should have in your
+extension:
```

Comment:
What about linking to the [file with this content](https://github.com/solidusio/solidus_support/blob/master/.circleci/config.yml) instead? That way, if we change something, we don't need to revise the doc as well.

Reply:
Good idea, but I'll link [this file](https://github.com/solidusio/solidus_dev_support/blob/master/lib/solidus_dev_support/templates/extension/.circleci/config.yml), since it's what is copied in extensions when creating them. Does it work?

Reply:
sure ❤️